municipal_finance/migrations/0024_delete_auditopinions.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-11-27 13:51
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('municipal_finance', '0023_capital_initial_data'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
```

Comment:
why?

Reply:
Wow! I've got no idea where this came from. Getting rid of it.

Reply:
Ah I forgot @jbothma! It was an unused model. We've got AuditOpinionFacts for the actual data. That good with you?

Reply:
aaah cool - I guess the build would have broken if this table was used but missing anyway